modules/page-memory/src/integrationTest/java/org/apache/ignite/internal/pagememory/persistence/ItBplusTreePageMemoryImplTest.java

```diff
@@ -55,7 +55,8 @@ protected PageMemory createPageMemory() throws Exception {
                 (page, fullPageId, pageMemoryEx) -> {
                 },
                 (fullPageId, buf, tag) -> {
-                }
+                },
+                PAGE_SIZE
```

Comment:
PageMemoryImpl has pageSize as a parameter, instead of PageMemoryConfiguration or something. Why?